src/workflows-app/components/SubmissionDetailsInputsTable.ts

```diff
@@ -0,0 +1,125 @@
+import { ReactNode, useState } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { AutoSizer } from 'react-virtualized';
+import { FlexTable, Sortable, tableHeight, TextCell } from 'src/components/table';
+import { getInputTableData, InputTableData } from 'src/workflows-app/utils/submission-utils';
+
+import { InputDefinition, RecordLookupInputSource } from '../models/submission-models';
+
+const rowWidth = 100;
+const rowHeight = 50;
+
+type SubmissionDetailsInputsTableProps = {
+  configuredInputDefinition: InputDefinition[];
+};
+
+const SubmissionDetailsInputsTable = ({ configuredInputDefinition }: SubmissionDetailsInputsTableProps): ReactNode => {
+  const [sort, setSort] = useState<{
+    field: 'taskName' | 'variable';
+    direction: 'asc' | 'desc';
+  }>({
+    field: 'taskName',
+    direction: 'asc',
+  });
+
+  const includeOptionalInputs = true;
+  const searchFilter = '';
+
+  const inputTableData: InputTableData[] = getInputTableData(
+    configuredInputDefinition,
+    searchFilter,
+    includeOptionalInputs,
+    sort
+  );
+
+  return div(
+    {
+      style: {
+        backgroundColor: 'rgb(235, 236, 238)',
+        display: 'flex',
+        flex: '1 1 auto',
+        flexDirection: 'column',
+        padding: '1rem 3rem',
+      },
+    },
+    [
+      div(
+        {
+          style: {
+            height: tableHeight({
+              actualRows: inputTableData.length,
+              maxRows: inputTableData.length,
```

Comment:
Yes. FlexTable uses a virtualized table so that it only renders visible rows. This allows performantly rendering large numbers of rows. https://web.dev/articles/virtualize-long-lists-react-window describes the concept in more detail (though that's not the exact library that Terra UI uses).

The general pattern is that you have some div that fills the remaining available space in the window (with a minimum height to work in small windows), then AutoSizer sizes the table's viewport based on that div.